wrapper/virt_v2v_wrapper.py

```diff
@@ -347,11 +352,26 @@ def main():
                 data, host.get_uid(), host.get_gid())
             if agent_pid is None:
                 raise RuntimeError('Failed to start ssh-agent')
+        source_host = detect_source_host(data, agent_sock)
+        if source_host:
+            try:
+                source_host.prepare_exports()
+            except Exception as e:
```

Comment:
Cleanup after all issues is already implemented and all cleanups should be inside `finish()`.

Reply:
This is here because close_exports needs to use SSH, but the SSH agent is killed before `finish()`. Can the agent be closed after (or inside of) `finish()`? Or maybe I can move the work done by close_exports into transfer_exports.

Reply:
My bad, not all finishes do clean-ups and it really depends. Since you need to do a cleanup in the other side, it is a different story. I think we can kill the ssh agent later as well. I just don't like the catch-all since there is already one. I think killing the agent later (why not in finish) is a safer choice.

Reply:
I had tried moving this and the extra branching into source_hosts: https://github.com/mrnold/manageiq-v2v-conversion_host/commit/a014df765c95037a2ed0c4178151922348e8ef3e

It makes virt_v2v_wrapper easier to read, and catches fewer exceptions. But maybe it really is better to count close_exports as a clean-up function, and move it into finish. I will try it that way and see how it looks.

Reply:
Thanks. Although this was not a requirement, just something I noticed could be done in a cleaner way, that's it. Whatever works for you will be fine.